lib/Graph/Nodes.cpp

```diff
@@ -199,7 +199,8 @@ static bool verifyPool(NodeValue src, NodeValue dest,
   ShapeNHWC exp(idim.n, outSz.first, outSz.second, idim.c);
   isValid &=
       expectCompareTrue("Unexpected output dimensions", exp, odim, parent);
-  isValid &= checkTypeIgnoreShape(src, dest, dest.getNode());
+  if (!src.getType()->isQuantizedType())
```

Comment:
The reason you added this is because pools may have different `scale`/`offset` for the `src` and `dest`, correct? Can you instead do `checkSameIsQuantized()` if `src` is quantized?